modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/java/com/liferay/frontend/editor/alloyeditor/web/internal/editor/configuration/BaseAlloyEditorConfigContributor.java

```diff
@@ -62,7 +62,7 @@ public void populateConfigJSONObject(
 			"disableNativeSpellChecker", Boolean.FALSE
 		).put(
 			"extraPlugins",
-			"ae_autolink,ae_dragresize,ae_addimages,ae_imagealignment," +
+			"addimages,ae_autolink,ae_dragresize,ae_imagealignment," +
```

Comment:
Now that you do this... I would love to see `ae_autolink` and maybe others moved as well 😁 

I'll create some tickets for it ❤️